src/commands/firestore-databases-create.ts

```diff
@@ -0,0 +1,60 @@
+import { Command } from "../command";
+import * as clc from "colorette";
+import * as fsi from "../firestore/api";
+import * as types from "../firestore/api-types";
+import { logger } from "../logger";
+import { requirePermissions } from "../requirePermissions";
+import { Emulators } from "../emulator/types";
+import { warnEmulatorNotSupported } from "../emulator/commandUtils";
+import { FirestoreOptions } from "../firestore/options";
+
+export const command = new Command("firestore:databases:create")
+  .description("Create a database in your Cloud Firestore project.")
+  .option(
+    "--pretty",
+    "Pretty print. When not specified the databases are printed in the " +
+      "JSON specification format."
+  )
+  .option("--database <databaseId>", "Name of database to be created. (required).")
+  .option(
+    "--location <locationId>",
+    "Region to create database, for example 'nam5'. Run 'firebase firestore:locations --pretty' to get a list of eligible locations. (mandatory)"
```

Comment:
Can we be consistent with "required" vs. "mandatory"?

Reply:
done